README.md

```diff
@@ -25,9 +25,9 @@ See official [API Docs](https://littlebee.github.io/basic_bot/)
 On your development computer (not your bot's onboard computer; see later in this doc)
 
 ```sh
-python3 -m venv my_new_project_venv
+python -m venv my_new_project_venv
 source my_new_project_venv/bin/activate
-python3 -m pip install git+https://github.com/littlebee/basic_bot.git@main
+python -m pip install git+https://github.com/littlebee/basic_bot.git@main
```

Comment:
Oh, and I forgot to include a mention of changing these in the human-generated description. I think this is the recommended way of using `pip` in a venv, but I don't really know.